### PR TITLE
Start of top-level documentation

### PR DESCRIPTION
**Pull request by rpellows**
Some links are TBD

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Maintainer
+Maintainers are responsible for the strategic interests of the component and
+the project. A Maintainer reviews code contributions and accepts pull requests,
+continuously grooms the backlog and balances the project's long-term goals
+while managing timely releases.  Maintainers take and provide feedback to
+improve existing processes and create new ones.
+
+**Learn More**
+
```
Comment (vjtirumalai) [inline docs/community.md:59]: Given the exercise we went through yesterday, we should add a hook/link to a page that describes the required project and tools setup permission process. I would envision a maintainer of a project would need to do this.

**Review by vjtirumalai — CHANGES_REQUESTED** [review]
LGTM! One small change requested.

**Review by mrmccarthy — APPROVED** [review]
Looks good to me.

**Comment by vjtirumalai** [comment]
As discussed plz add a owners file also and add yours, mine and mike's name to both reviewers and maintainers to begin with

**Review by vjtirumalai — APPROVED** [review]
Thanks for the changes!

**Pull request merged**